app/policies/user_invited_event_policy.rb

```diff
@@ -0,0 +1,20 @@
+class UserInvitedEventPolicy < ApplicationPolicy
```

Comment:
This shouldn't be on the `UserInvitedEvent`, that's just a audit event, not the actual action